docs/manual/docs/examples/graph/call-graph.md

```diff
@@ -0,0 +1,24 @@
+# Authenticate with and call the Microsoft Graph 
+
+Author: [Garry Trinder](https://github.com/garrytrinder)
+
+Obtain a new access token for the Microsoft Graph and use it an HTTP request.
+
+```powershell tab="PowerShell Core"
+$token = o365 accesstoken get --resource https://graph.microsoft.com --new
+$me = Invoke-RestMethod -Uri https://graph.microsoft.com/v1.0/me -Headers @{"Authorization"="Bearer $token"}
+$me
+```
+
+```bash tab="Bash"
```

Comment:
We should include a remark that this script requires jq, like we do in other scripts